### ISSUE TITLE
conda-execute and conda 4.1.0 compat issue

### ISSUE THREAD

**Issue by jakirkham** [pelson/conda-execute#21]
Seems there is a break with `conda-execute` and `conda`. See the traceback below. This uses `conda` 4.1.0 and `conda-env` 2.5.0. See this [log]( https://travis-ci.org/conda-forge/conda-forge.github.io/jobs/137709725 ) for details.

```
Traceback (most recent call last):
  File "/home/travis/conda/bin/conda-execute", line 9, in <module>
    load_entry_point('conda-execute==0.4.1', 'console_scripts', 'conda-execute')()
  File "/home/travis/conda/lib/python3.5/site-packages/conda_execute/execute.py", line 177, in main
    exit(execute(path, force_env=args.force_env, arguments=args.remaining_args))
  File "/home/travis/conda/lib/python3.5/site-packages/conda_execute/execute.py", line 84, in execute
    env_prefix = create_env(env_spec, force_env, spec.get('channels', []))
  File "/home/travis/conda/lib/python3.5/site-packages/conda_execute/tmpenv.py", line 87, in create_env
    if not conda.install.is_extracted(conda_execute.config.pkg_dir, dist_name):
TypeError: is_extracted() takes 1 positional argument but 2 were given
```

cc @msarahan

**Comment by pelson** [comment]
Thanks @jakirkham. Should be fixed in #23.

**Comment by jakirkham** [comment]
...or PR ( https://github.com/pelson/conda-execute/pull/22 )? 😕

In any event, thanks for the fix. 😄

**Comment by pelson** [comment]
Fill be fixed in v0.6.0.
